beers/management/commands/importbjcp.py

```diff
@@ -0,0 +1,109 @@
+import urllib.request
+import xml.etree.ElementTree
+
+from django.core.management.base import BaseCommand, CommandError
+
+from beers.models import BeerStyle
+
+
+def parse_subcategory(element):
+    """Parse subcategory element of styleguide xml."""
+    subcategory = {}
+    subcategory_id = element.get('id')
+
+    if subcategory_id[0] in ['C', 'M']:
+        category_id = subcategory_id[0:2]
+        subcat_letter = subcategory_id[2]
+    else:
+        category_id = subcategory_id[0]
+        subcat_letter = subcategory_id[1]
+
+    subcategory['category'] = category_id
+    subcategory['subcategory'] = subcat_letter
+
+    for key in ['name', 'aroma', 'appearance', 'flavor', 'mouthfeel',
+                'impression', 'comments', 'history', 'ingredients',
+                'comparison', 'examples', 'tags']:
+        val = element.find(key)
+        if val is not None:
+            subcategory[key] = val.text
+
+    stats = element.find('stats')
+    if stats is not None:
+        for key in ['ibu', 'og', 'fg', 'srm', 'abv']:
+            stat = stats.find(key)
+            if stat is not None:
+                if stat.find('low') is not None:
+                    subcategory[key + '_low'] = float(stat.find('low').text)
```

Comment:
Since the fields are now decimal fields, use `decimal.Decimal(stat.find('low').text)` instead.